blocks/copilot/ims.js

```diff
@@ -1,8 +1,8 @@
-function getAccessToken(cfg = {}) {
-	if(!cfg.token) {
-		//call IMS API
-	}
-	return cfg.token;
+export default function getAccessToken(cfg = {}) {
+  if (!cfg.token) {
```

Comment:
should be add the ims api code also if some one wants to use the instance later?

Reply:
Ideally yes, though probably for demo we can just use the hardcoded token.